packages/checkout/sdk/src/smartCheckout/routing/routingCalculator.ts

```diff
@@ -226,150 +257,107 @@ export const routingCalculator = async (
   const allowList = await measureAsyncExecution<RoutingTokensAllowList>(
     config,
     'Time to get routing allowlist',
-    allowListCheck(
-      config,
-      tokenBalances,
-      availableRoutingOptions,
-    ),
+    allowListCheck(config, tokenBalances, availableRoutingOptions),
   );
 
   // Ensures only 1 balance requirement is insufficient
   const insufficientRequirement = getInsufficientRequirement(balanceRequirements);
 
   const routePromises: Promise<any>[] = [];
-  const routeSteps: string[] = [];
+  const fundingRoutes: FundingRoute[] = [];
+  let priority = 0;
 
-  if (availableRoutingOptions.bridge) {
-    routeSteps.push('bridgeFundingStep');
-    routePromises.push(getBridgeFundingStep(
-      config,
-      readOnlyProviders,
-      availableRoutingOptions,
-      insufficientRequirement,
-      tokenBalances,
-    ));
+  if (availableRoutingOptions.swap) {
+    routePromises.push(
+      getSwapFundingSteps(
+        config,
+        availableRoutingOptions,
+        insufficientRequirement,
+        ownerAddress,
+        tokenBalances,
+        allowList.swap,
+        balanceRequirements,
+      ).then((result) => {
+        if (result.length) {
+          priority++;
```

Comment:
@mimi-imtbl I think priorities need to be fixed numbers, otherwise the priority will change depending on which promise resolves first.

So you may better just assign a number
swap: 1
bridge: 2
onramp: 3
bridge&swap: 4

Reply:
The reason I didn't change it to fixed numbers is because in case `swap` routes are not found, then `bridge` routes will get the `priority:1`, so `bridge` routes are not always having `priority:2`

Reply:
@dreamoftrees do you think the final funding routes should have priorities that are strictly starting from 1? Otherwise, we can hardcoded them as J mentioned above 👀

Reply:
@mimi-imtbl you are correct, yet whats relevant here is that priority is only used to sort the results, so as long as there is a sequence the actual numbers to matter. 1, 2, 3 is equal to 1 3 4

Reply:
sounds good

Reply:
Sorry just catching up here. Sounds like you guys have it sorted.. as in bring able to `sort` I think is more important than strictly starting from 1.